dragonfly-client/src/proxy/mod.rs

```
@@ -595,6 +636,25 @@ async fn proxy_by_dfdaemon(
         }
     };
 
+    // Get the content from the cache by the request.
+    match cache.get_by_request(&download_task_request).await {
+        Ok(None) => {
+            debug!("cache miss");
+        }
+        Ok(Some(content)) => {
+            debug!("cache hit");
+            let body_boxed = Full::new(content).map_err(ClientError::from).boxed();
+            return Ok(Response::new(body_boxed));
+        }
+        Err(err) => {
+            error!("get content from cache failed: {}", err);
```

why not ignore this error? could we continue if has err when get cache?

---

Rocksdb error needs to fail fast.